jest-integration-config.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-undef */
+module.exports = {
+  displayName: 'Integration Tests',
+  testEnvironment: 'node',
+  testMatch: ['**/src/tests/*.test.ts'],
```

Comment:
Os testes estarão no próprio diretório do módulo/componente/página, não?
Se for o caso, esse caminho está incorreto.